services/thumbnail_queue/thumbnail_queue.py

```diff
@@ -26,118 +21,115 @@
 
 queue = []
 
+THUMBNAIL_TYPES = ["sdss", "ls", "ps1"]
+
+REQUEST_TIMEOUT_SECONDS = cfg['health_monitor.request_timeout_seconds']
 
-def service(queue):
+host = f'{cfg["server.protocol"]}://{cfg["server.host"]}' + (
+    f':{cfg["server.port"]}' if cfg['server.port'] not in [80, 443] else ''
+)
 
+
+def is_loaded():
```

Comment:
since we use this over and over again, could it be put into a microservices utils file?

Reply:
Done!